miso-web/src/main/webapp/scripts/run_ajax.js

```diff
@@ -22,6 +22,8 @@
  */
 
 var Run = Run || {
+  userIsAdmin: false,
+
   deleteRun: function(runId, successfunc) {
     if (confirm("Are you sure you really want to delete RUN" + runId + "? This operation is permanent!")) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this usage of confirm(...). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1442)